openmmtools/multistate/utils.py

```diff
@@ -177,7 +181,7 @@ def get_equilibration_data_per_sample(timeseries_to_analyze, fast=True, max_subs
         except:
             g_i[i] = (time_size - t + 1)
         n_effective_i[i] = (time_size - t + 1) / g_i[i]
-    return i_t, g_i, n_effective_i
+    return i_t[1:], g_i[1:], n_effective_i[1:] # We should never choose data from the first time origin as the equilibrated data
```

Comment:
Could we add "because it contains snapshots warming up from minimization, which causes problems with correlation time detection"?

Reply:
Done